Add / Delete Pages in the Launcher

From: Rod Whitby <rod@whitby.id.au>

http://predev.wikidot.com/add-delete-pages-in-the-launcher

Original announcement in #webos-internals IRC channel:

Jun 11 15:33:00 <jwsh> I put some instructions on how to enable adding pages to the launcher on the wiki
---

 .../app/controllers/launcher-assistant.js          |    6 ++----
 1 files changed, 2 insertions(+), 4 deletions(-)


diff --git a/usr/lib/luna/system/luna-applauncher/app/controllers/launcher-assistant.js b/usr/lib/luna/system/luna-applauncher/app/controllers/launcher-assistant.js
index 79a2cc5..7bf45b5 100644
--- a/usr/lib/luna/system/luna-applauncher/app/controllers/launcher-assistant.js
+++ b/usr/lib/luna/system/luna-applauncher/app/controllers/launcher-assistant.js
@@ -28,8 +28,8 @@ var LauncherAssistant = Class.create({
 		visible: true,
 		label: $L('Launcher'),
         items: [
-			/*{ label: $L('New page'), command: 'newpage' },
-			{ label: $L('Delete page'), command: 'deletepage' },*/
+			{ label: $L('New page'), command: 'newpage' },
+			{ label: $L('Delete page'), command: 'deletepage' },
 			Mojo.Menu.editItem,
 			{ label: $L('List Apps...'), command: 'listapps' },
 			Mojo.Menu.helpItem
@@ -122,7 +122,6 @@ var LauncherAssistant = Class.create({
 				case 'listapps':
 					ApplicationService.launch(this.deviceInfo.id, this.deviceInfo.params);
 					break;
-				/*
 				case 'newpage':
 					if (this.pageDivs.length < 10) {
 						this.insertPage(this.activePageIndex, true);
@@ -136,7 +135,6 @@ var LauncherAssistant = Class.create({
 						}
 					}
 					break;
-				*/
 			}
 		}
 	},
